MaxText/configs/experimental/1024b.sh

```diff
@@ -16,13 +16,13 @@ done
 
 if [[ $PLATFORM == "gce" ]]; then
```

Comment:
Could you change this if-else block to just `bash preflight.sh PLATFORM=$PLATFORM`? I think I forgot to update it before

Reply:
I am not sure how to change the DNS lookup for GCE and GKE next:

```
# For DNS lookup when running on large number of VMs on gce
    echo '142.250.123.95 www.googleapis.com' | sudo tee -a /etc/hosts
    echo '142.251.4.128 storage.googleapis.com' | sudo tee -a /etc/hosts
```

```
 # For DNS lookup when running on large number of VMs on gke
    echo '142.250.123.95 www.googleapis.com' | tee -a /etc/hosts
    echo '142.251.4.128 storage.googleapis.com' | tee -a /etc/hosts
```

Do you mind add  `preflight.sh` in the next PR and I can help review.

Reply:
You can remove those, we don't need them anymore.

Reply:
Done.